index.html

```diff
@@ -70,11 +65,10 @@ <h5 class="venue">
     <!-- main program section -->
     <section class="main-program">
       <h2 class="main-title">Main Program</h2>
-      <img src="#" alt="" />
       <div class="mainCard">
         <div class="cardA">
           <div class="card-img">
-            <img src="image/mpro-img1-removebg-preview.png" alt="" />
+            <img src="image/mpro-img1-removebg-preview.png" alt="image" />
```

Comment:
- [ ] This issue is not fixed yet, please take a few more fixtures for it.

FIGMA DESIGN
<img width="284" alt="1" src="https://user-images.githubusercontent.com/77843050/227719449-e3a73514-645d-4c95-9ca0-de6ebf555892.png">

YOUR DESIGN
<img width="665" alt="2" src="https://user-images.githubusercontent.com/77843050/227719476-b705455d-bcff-401c-a844-fcf8f307495d.png">